sfs/mono/wfs.py

```diff
@@ -217,6 +234,92 @@ def point_25d(omega, x0, n0, xs, xref=[0, 0, 0], c=None, omalias=None):
     return d, selection, secondary_source_point(omega, c)
 
 
+def point_25d(omega, x0, n0, xs, xref=[0, 0, 0], c=None, omalias=None):
+    r"""Driving function for 2.5-dimensional WFS of a virtual point source.
+
```

Comment:
Here you should use the `versionchanged` directive.

Reply:
fixed